### ISSUE TITLE
Coffeescript

### ISSUE THREAD

**Issue by jakee246** [jakee246/tc359finalproject#2]
Random quote generator using coffeescript.

**Comment by chrisvfritz** [comment]
I just want to confirm that it looks like you didn't finish this one, as it doesn't look like the `getQuote` function you defined doesn't actually get a quote yet. Does that sound right to you? Just want to make sure I didn't miss something.

**Comment by jakee246** [comment]
Correct, I was unable to complete it. I could not get the process to work
for me.

**Comment by chrisvfritz** [comment]
Got it. In case you're still curious about CoffeeScript, here's [a working version](https://github.com/PStanos/tc359_rails_final/blob/master/app/assets/javascripts/joke_generator.coffee) that another student made and [this is a refactor I made](https://github.com/PStanos/tc359_rails_final/issues/1#issuecomment-91244467) to take better advantage of CoffeeScript features.
